src/components/containers/css-grid.js

```diff
@@ -7,22 +7,26 @@ const CssGrid = styled.div`
     height: ${props => props.height || '100%'};
     margin: ${props => props.margin || '0'};
     grid-template-columns: ${props => props.columns || 'auto'};
-    grid-column-gap: ${props => props.columngap || '0'};
-    grid-row-gap: ${props => props.rowgap || '0'};
+    grid-column-gap: ${props => props.column_gap || ''};
+    grid-row-gap: ${props => props.row_gap || ''};
```

Comment:
Homepage is using the `<CssGrid />` also now so please update the prop names there too